client/layout/sidebar/expandable.jsx

```diff
@@ -98,7 +99,7 @@ export const ExpandableSidebarMenu = ( {
 		setSubmenuHovered( false );
 	};
 
-	const menuId = uniqueId( 'menu' );
+	const menuId = 'menu' + uuid().replace( /-/g, '' );
```

Comment:
Does this really create a new ID on every re-render? That looks like a (pre-existing) bug. It should be allocated once per instance. `useMemo( () => uuid(), [] )`.

Reply:
👍  Good catch!

I'm fixing it here as I'm touching this very line. Fixed in https://github.com/Automattic/wp-calypso/pull/52673/commits/65f4afee639d34de97ee3bdf5229cc7bdf1fef85